modules/javafx.graphics/src/main/java/javafx/scene/layout/Background.java

```diff
@@ -353,6 +353,19 @@ public Background(final @NamedArg("fills") BackgroundFill[] fills, final @NamedA
         hash = result;
     }
 
+    /**
+     * A convenience factory method for creating a background with a single {@code Paint}. The background will use the
```

Comment:
Since th `Background` constructors take a list of `Paint` objects, I think saying a "single" `{@code Paint}` is helpful. I can see how adding "for `BackgroundFill`" (or maybe "as a `BackgroundFill`"?) might make it clearer.

Reply:
Sound good to me, It would read like `A convenience factory method for creating a Background by specifying a single {@code Paint} as a BackgroundFill`

Reply:
Does mentioning `BackgroundFill` in the description contribute something? The `@implSpec` tag gives that info. In my eyes, if the docs say the background will use a single `Paint` then I know exactly what I'm going to get regardless of what it uses in the implementation.